src/target/riscv/riscv-013.c

```diff
@@ -990,9 +1005,10 @@ static int register_read_abstract_with_size(struct target *target,
 	uint32_t command = access_register_command(target, number, size,
 			AC_ACCESS_REGISTER_TRANSFER);
 
-	int result = execute_abstract_command(target, command);
+	unsigned int abstractcs_err;
+	int result = execute_abstract_command(target, command, &abstractcs_err);
```

Comment:
The third parameter of execute_abstract_command() is called `cmderr`, so I'd recommend to name the argument also `cmderr` (in order to not introduce different names for the same thing).

```suggestion
	unsigned int cmderr;
	int result = execute_abstract_command(target, command, &cmderr);
```

Reply:
addressed